README.md

```diff
@@ -1,17 +1,35 @@
-# CircleCI Orb: GitHub PR [![CircleCI Orb Version](https://img.shields.io/badge/endpoint.svg?url=https://badges.circleci.io/orb/narrativescience/ghpr)](https://circleci.com/orbs/registry/orb/narrativescience/ghpr) [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+# CircleCI Orb: GitHub PR
 
-[View in CircleCI Orb Registry](https://circleci.com/orbs/registry/orb/narrativescience/ghpr)
+[![CircleCI Orb Version](https://img.shields.io/badge/endpoint.svg?url=https://badges.circleci.io/orb/narrativescience/ghpr)](https://circleci.com/orbs/registry/orb/narrativescience/ghpr) [![License](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
-Set of git utilities to manage GitHub Pull Requests in CI, namely simulating the result of
-merging the head branch into the PR's target base branch.
+Set of git utilities to manage GitHub Pull Requests in CI. This orb was created to address the need to simulate the result of merging the head branch into a PR's target base branch.
 
-To use, create a service GitHub user with at least Read access to your repository and set the following environment variables in a [Context](https://circleci.com/docs/2.0/contexts/) or your [Project's Environment Variables](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project):
+Additional features:
 
-* `GITHUB_USERNAME` - Username of the service user that has read/write permissions to the repo.
-* `GITHUB_PASSWORD` - Password of the service user.
-* `GITHUB_EMAIL` - Email of the service user.
+- Posting PR comments on success/failure/always
+- Sending Slack notifications to PR authors
 
 The commands in the orb will expose the following environment variables:
 
 * `GITHUB_PR_BASE_BRANCH` - The base branch for the PR.
 * `GITHUB_PR_NUMBER` - The number of the PR.
+
+## Getting Started
+
+To use this orb, there are a few environment variables to set. These can be set in a [Context](https://circleci.com/docs/2.0/contexts/) or your [Project's Environment Variables](https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project).
```

Comment:
I normally try to limit line length of readmes to ~80. Obviously tough to do with long links like this but I find it makes editing easier and forces you to be more conscious of how much you're including in a paragraph. What do you think?

Reply:
Also, thanks for adding these links, that's really helpful!